tap_pipedrive/tap.py

```diff
@@ -215,18 +216,29 @@ def execute_stream_request(self, stream):
         return self.execute_request(stream.endpoint, params=params)
 
     def execute_request(self, endpoint, params=None):
-        headers = {
-            'User-Agent': self.config['user-agent']
-        }
-        _params = {
-            'api_token': self.config['api_token'],
-        }
+        # Using bearer token generated via OAuth authentication
+        if 'access_token' in self.config:
+            headers = {
+                'User-Agent': self.config['user-agent'],
+                'Authorization': 'Bearer ' + self.config['access_token']
+            }
+            _params={}
+            url = "{}/{}".format(BASE_OAUTH_URL, endpoint)
+            logger.debug('Firing request at {} with ouath:'.format(url))
```

Comment:
Tiny typo
```suggestion
            logger.debug('Firing request at {} with OAuth:'.format(url))
```

Reply:
Hi @luandy64, Thanks for your inputs! I addressed your comments, pls see updated PR.